infra/k8s/README.md

```diff
@@ -56,9 +56,9 @@ export ZONES=eu-central-1a
 kops create cluster \
   --zones $ZONES \
   --master-zones $ZONES \
-  --master-size m4.xlarge \
-  --node-size m4.xlarge \
-  --node-count 2 \
+  --master-size c5.2xlarge \
```

Comment:
Looks like our setup is CPU intensive, so it makes more sense to run `c5` instances, rather than `m4` or `m5` for now.